packages/kbn-i18n/types/intl_format_cache.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module 'intl-format-cache' {
```

Comment:
__nit:__ please add license header manually to typings files - there is a bug in prettier/tslint that skip declaration files and hence we don't have license header in them.